src/misc/ssEconomy.lua

```diff
@@ -180,6 +184,8 @@ function ssEconomy:loadGameFinished()
     -- Some calculations to make the code faster on the hotpath
     ssEconomy.aiPricePerMSWork = ssEconomy.aiPricePerHourWork / (60 * 60 * 1000)
     ssEconomy.aiPricePerMSOverwork = ssEconomy.aiPricePerHourOverwork / (60 * 60 * 1000)
+    local vanillaPricePerHour = 2000
+    ssEconomy.aiVanillaToSeasonsMultiplier = ssEconomy.aiPricePerHourWork / vanillaPricePerHour
```

Comment:
Did this due to the [Coding Conventions & Practices](https://github.com/RealismusModding/FS17_seasons/wiki/Coding-Conventions-&-Practices) section on the wiki, regarding magic numbers.